tealfmt.go

```diff
@@ -11,165 +11,158 @@ import (
 	"github.com/docopt/docopt-go"
 )
 
-const version = "tealfmt v0.1.0"
-const usage = `tealfmt
+const (
+	version = "tealfmt v0.1.0"
+	usage   = `tealfmt
 
 Usage:
-  tealfmt <file> [ -e | --edit ]
+  tealfmt <file> [ -i | --inplace ]
   tealfmt -h | --help
   tealfmt --version
 
 Options:
   -h --help     Show this screen.
   --version     Show version.
-  -e --edit     Edit the file in place.`
-
-var config struct {
-	File string
-	Edit bool
-}
-
-var voidOps = [...]string{"assert", "app_global_put", "b", "bnz", "bz", "store",
-	"stores", "app_local_put", "app_global_del", "app_local_del", "callsub",
-	"log", "itxn_submit", "itxn_next"}
-
-func addEmptyLine(newLinesPtr *[]string, emptyCheckLine string) {
-	newLines := *newLinesPtr
+  -i --inplace  Edit the file in place.`
+)
 
-	if emptyCheckLine != "" {
-		newLines = append(newLines, "")
+var (
+	voidOps = []string{
+		"assert", "err", "return", "app_global_put", "b", "bnz", "bz", "store",
+		"stores", "app_local_put", "app_global_del", "app_local_del", "callsub",
+		"log", "itxn_submit", "itxn_next",
 	}
 
-	*newLinesPtr = newLines
-}
-
-func contains(strs []string, str string) bool {
-	for _, ss := range strs {
-		if ss == str {
-			return true
-		}
-	}
-	return false
-}
+	versionMatch = regexp.MustCompile(`^#pragma `)
+	opMatch      = regexp.MustCompile(`\S+`)
+	labelRegex   = regexp.MustCompile(`\S+:($| //)`)
+	commentRegex = regexp.MustCompile(`^//`)
+	voidOpMatch  = regexp.MustCompile("^(" + strings.Join(voidOps, "|") + ")$")
```

Comment:
The regex should end with `($| //)` so void ops followed by a comment are properly matched. 

I suppose to cover all cases it should really end with `($|\s*//)`

Reply:
I pass the `op` in not the full line so I think this should still work?